testdata/values.txt

```diff
@@ -0,0 +1,30 @@
+mito -use collections,try src.cel
+! stderr .
+cmp stdout want.txt
+
+-- src.cel --
+{
+	"good_instance": {"a": ["aa", "ab"], "b": [2, 3], "c": {"cc": 4, "cd": 5}}.values(),
+	"good_function": values({1: "a", 2: "b"}),
+}
+-- want.txt --
+{
+	"good_function": [
+		"a",
+		"b"
+	],
+	"good_instance": [
+		[
+			"aa",
+			"ab"
+		],
+		[
+			2,
+			3
+		],
+		{
+			"cc": 4,
+			"cd": 5
+		}
+	]
```

Comment:
CI is failing because map doesn't maintain order when iterating with `range`. Any workarounds to make the tests pass?

Reply:
I think a nice property to have would be to make `zip(map_keys(m), map_values(m))` return the original map (well, a copy of it). This currently does not happen, but if it did, this test would pass. It adds an allocation cost to the `map_values` call and a sort CPU cost to both, but I think it is worth it.

Reply:
Added new test into zip to test `zip(map_keys(m), map_values(m))`